pinc/abort.inc

```diff
@@ -122,7 +122,7 @@ function dump_request_info()
                 break;
             default:
```

Comment:
In cases like this, I think we should remove the assert entirely.

Reply:
Or simplify it radically. You can do variable lookup by string with `$array = ${$array_name};`